Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_001.xml

```diff
@@ -0,0 +1,374 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
> is it your intention to include all RP_Detector_Assembly_* files in one geometry scenario or have multiple scenarios per file? Note, the default geometry scenario will be picked up from a DB and cannot be modified on the fly.

We plan to always use all RP_Detector_Assembly_* files - each of them corresponds to one RP which physically exists. The only thing which can/will change run to run is the position of the RPs wrt. beam.